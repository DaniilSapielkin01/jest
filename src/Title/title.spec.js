import React from 'react';
import Title from './title';

describe("Title component", () => { 

    it("should render Title with props" , () => { 
        const component = shallow(<Title title={"Test title #1"} />);
        expect(component).toMatchSnapshot();
    });

    it("should render Title with props" , () => { 
        const component = shallow(<Title />);
        expect(component).toMatchSnapshot();
    });

});