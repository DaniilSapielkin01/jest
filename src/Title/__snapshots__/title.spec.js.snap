// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render Title with props 1`] = `
<h1
  className="title"
>
  Test title #1
</h1>
`;

exports[`Title component should render Title with props 2`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
